guides/17_Migration_to_3_0.md

```diff
@@ -0,0 +1,125 @@
+# Migration from SDK v1 to SDK v2
+
+## Installation
+
+Installation involves adjusting your `package.json` to depend on version `3.0.0`.
+
+```shell
+npm install --save amazon-chime-sdk-js@3
+```
+
+## Interface changes
+
+Version 3 of the Amazon Chime SDK for JavaScript makes a number of interface
+changes.
+
+In many cases you should not need to adjust your application code at all. This will be the case if:
+
+* You do not implement your own `EventController` or construct `DefaultEventController` yourself.
+* You do not pass a `EventReporter` yourself or use the reporter through `MeetingSession`.
+* You do not explicitly call `addObserver` on any instances of `AudioVideoController` using a `AudioVideoObserver` that implements `eventDidReceive`.
+
+If your application does not meet all four criteria, read on.
+
+### Updating `EventController` use cases
+The `EventController` has been decoupled from the `AudioVideoController` which has added addtional required functions to the interface.
+```typescript
+addObserver(observer: EventObserver): void;
+removeObserver(observer: EventObserver): void;
+forEachObserver(observerFunc: (observer: EventObserver) => void): void;
+// Optional
+
+```
+
+Addtionally because of this the constructor of `DefaultEventController` has been updated
+```typescript
+// Old Syntax
+constructor(
+    audioVideoController: AudioVideoController
+    eventReporter?: EventReporter
+  )
+
+// New Syntax
+constructor(
+    configuration: MeetingSessionConfiguration,
+    logger: Logger,
+    eventReporter?: EventReporter
+  )
+```
+
+### Updating `EventReporter` use cases
+
+The `DefaultMeetingSession` constructor no longer takes in a `EventReporter` as it is no longer in charge constructing a reporter.
```

Comment:
`in charge *of* constructing a reporter.`

Also i prefer `is no longer responsible for`

Reply:
Yeah I can just remove it, I wanted to add a bit of context as to why we're removing it from the constructor, but with the context from decoupling the event controller it is a bit redundant.